methods/coseismic/Coseismic_Requirement_Validation.ipynb

```diff
@@ -96,11 +96,11 @@
     "- [6.3. Amplitude vs. Distance of Relative Measurements (pair differences)](#M2ampvsdist2)\n",
```

Comment:
<p>@nabolfath Can you change this to:</p><pre class="ql-syntax" spellcheck="false">print('\n site&nbsp;&nbsp;disp-los [cm]')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>